soh/src/overlays/actors/ovl_En_Skj/z_en_skj.c

```diff
@@ -1567,32 +1564,29 @@ void EnSkj_WaitToGiveReward(EnSkj* this, PlayState* play) {
 }
 
 void EnSkj_GiveOcarinaGameReward(EnSkj* this, PlayState* play) {
-    if (Actor_HasParent(&this->actor, play)) {
+    if (Actor_HasParent(&this->actor, play) || (!GameInteractor_Should(GI_VB_GIVE_ITEM_FROM_OCARINA_MEMORY_GAME, true, this) && gSaveContext.ocarinaGameRoundNum != 3)) {
```

Comment:
I think we want the `gSaveContext.ocarinaGameRoundNum != 3` block within the vanilla behavior condition override for rando. Other mods might not want this, seems arbitrary.